services/app-web/src/pages/GraphQLExplorer/GraphQLExplorer.tsx

```diff
@@ -0,0 +1,144 @@
+import { ApolloExplorer } from '@apollo/explorer/react'
+import styles from './GraphQLExplorer.module.scss'
+import { useQuery, gql } from '@apollo/client'
+import { Loading } from '../../components'
+import { useAuth } from '../../contexts/AuthContext'
+import { GenericErrorPage } from '../Errors/GenericErrorPage'
+
+const INTROSPECTION_QUERY = gql`
```

Comment:
I could not figure out how to get our graphql schema as a string to pass into the `schema` prop in the `ApolloExplorer` component. I saw it could also accept an `IntrospectionQuery` type and did some googling to find out it was a query to get the entire schema. 

For query string itself is from the Apollo Explorer tool when using it from the browser. Since it was able to figure out our schema by just connecting to it, I figured it did an introspection query. Looked at the network tab and copied the introspection query it sent.

I could use some ideas on where to put this thing.